Extensions/TweenBehavior/JsExtension.js

```diff
@@ -37,6 +37,65 @@ module.exports = {
       )
       .setExtensionHelpPath('/behaviors/tween');
 
+      const easingChoices = JSON.stringify([
+        'linear',
+        'easeInQuad',
+        'easeOutQuad',
+        'easeInOutQuad',
+        'easeInCubic',
+        'easeOutCubic',
+        'easeInOutCubic',
+        'easeInQuart',
+        'easeOutQuart',
+        'easeInOutQuart',
+        'easeInQuint',
+        'easeOutQuint',
+        'easeInOutQuint',
+        'easeInSine',
+        'easeOutSine',
+        'easeInOutSine',
+        'easeInExpo',
+        'easeOutExpo',
+        'easeInOutExpo',
+        'easeInCirc',
+        'easeOutCirc',
+        'easeInOutCirc',
+        'easeOutBounce',
+        'easeInBack',
+        'easeOutBack',
+        'easeInOutBack',
+        'elastic',
+        'swingFromTo',
+        'swingFrom',
+        'swingTo',
+        'bounce',
+        'bouncePast',
+        'easeFromTo',
+        'easeFrom',
+        'easeTo',
+      ]);
+  
+      extension
+      .addExpression(
+        'Ease',
+        _('Ease'),
+        _('Tween between 2 values according to an easing function.'),
+        '',
+        'res/actions/camera.png'
+      )
+      .addParameter('expression', _('From value'))
+      .addParameter('expression', _('To value'))
+      .addParameter('stringWithSelector', _('Easing'), easingChoices, false)
```

Comment:
I think this should be the first or the last parameter? The first parameter would maybe make more sense (in a language that would allow "currying" of functions, you could create specific function by fixing the first parameter, and getting in return a function with 3 parameters: from, to, weight).

Also it reads a bit more nicely `Ease("elastic", 40, 100, 0.4)` in English : "Ease, using a elastic easing, from 40 to 100 with weight 0.4", because to the eye the parameter is almost part of the function name "Ease".